.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+data_file = .coverage
+omit =
+    checkin2015/settings*
+    *wsgi.py
+    *migrations*
+    *admin.py
+    *static*
+    *templates*
+
```

Comment:
my philosophy has been to cover test code, in part so you get the fixtures that you want to be testing, and in part to ensure you're running all tests. But I'm not firm on that position...I can add a filter for it.